index.html

```diff
@@ -2,31 +2,31 @@
 <!--//**********//PET FOOD EXERCISE//**********-->
 <!DOCTYPE html>
 <html>
-<head>
-	<meta charset="utf-8">
-	<meta http-equiv="X-UA-Compatible" content="IE=edge">
-	<title>Pet Food</title>
-	<link rel="stylesheet" href="css/main.css">
-</head>
-<body>
-	<!-- HEADER -->
-	<div id="header">
-		<!-- MAIN TITLE -->
-		<li><h1>Pet Food</h1></li>
-		<!-- NAV MENU -->
-		<div id="nav">
-			<ul>
-				<li>Products</li>
-				<li>About</li>
-			</ul>
-			<div id="blank"></div>
+	<head>
+		<meta charset="utf-8">
+		<meta http-equiv="X-UA-Compatible" content="IE=edge">
+		<title>Pet Food</title>
+		<link rel="stylesheet" href="css/main.css">
+	</head>
+	<body>
+		<!-- HEADER -->
+		<div id="header">
+			<!-- MAIN TITLE -->
+			<li><h1>Pet Food</h1></li>
```

Comment:
```li``` tags can only appear as a direct child of ```ul``` or ```ol``` tags.

Everything else looks awesome, feel free to merge in once you've pulled out those ```li```s 👍